__tests__/nc-news.test.js

```diff
@@ -32,12 +32,43 @@ describe("GET - /api/topics", () => {
         });
       });
   });
-  it("404 - GET /api/wrongEndpoint - should respond with a 404 status code when the endpoint does not exist", () => {
+});
+describe("GET - /api", () => {
+  it("should serve up a json representation of all the available endpoints of the api", () => {
     return request(app)
-      .get("/api/wrongEndpoint")
-      .expect(404)
+      .get("/api")
+      .expect(200)
       .then((response) => {
-        expect(response.body.msg).toBe("Route not found!");
+        expect(response.body.api).toEqual({
```

Comment:
This test will fail with every new endpoint. The json file can be required in as an object, how could you make this test more dynamic with that object?